InvenTree/InvenTree/urls.py

```diff
@@ -210,7 +211,7 @@
 new_frontendpatterns = [
     # Platform urls
     re_path(r'^platform/', include(platform_urls)),
-
+    re_path(r'^platform', spa_view, name='platform'),
```

Comment:
Are we going to hide this behind "experimental"?

Reply:
@SchrodingersGat it is enabled by default but can be controlled by config - `INVENTREE_PLATFORM_FRONTEND`